OpenShift: Annoying Random Issue when deploying into OpenShift

## Describe the bug

The deployment to OpenShift is randomly failing due to this exception:

```
12:49:34.706 Failed to execute goal io.quarkus:quarkus-maven-plugin:1.13.2.Final:build (default) on project config: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
12:49:34.706 	[error]: Build step io.quarkus.container.image.openshift.deployment.OpenshiftProcessor#openshiftBuildFromJar threw an exception: java.lang.RuntimeException: Execution of openshift build failed. See build output for more details
12:49:34.706 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftException(OpenshiftProcessor.java:513)
12:49:34.707 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild(OpenshiftProcessor.java:457)
12:49:34.707 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.lambda$openshiftBuild$10(OpenshiftProcessor.java:410)
12:49:34.707 	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
12:49:34.707 	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
12:49:34.708 	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
12:49:34.708 	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655)
12:49:34.708 	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
12:49:34.708 	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
12:49:34.708 	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
12:49:34.708 	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
12:49:34.709 	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
12:49:34.710 	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
12:49:34.710 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild(OpenshiftProcessor.java:410)
12:49:34.710 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.createContainerImage(OpenshiftProcessor.java:352)
12:49:34.710 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuildFromJar(OpenshiftProcessor.java:259)
12:49:34.710 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
12:49:34.711 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
12:49:34.711 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
12:49:34.711 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
12:49:34.711 	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:920)
12:49:34.711 	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
12:49:34.712 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2415)
12:49:34.712 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1452)
12:49:34.712 	at java.base/java.lang.Thread.run(Thread.java:834)
12:49:34.712 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
12:49:34.712 Caused by: java.io.IOException: Pipe closed
12:49:34.712 	at java.base/java.io.PipedInputStream.read(PipedInputStream.java:307)
12:49:34.712 	at java.base/java.io.PipedInputStream.read(PipedInputStream.java:377)
12:49:34.713 	at java.base/sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
12:49:34.713 	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
12:49:34.713 	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
12:49:34.713 	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:185)
12:49:34.713 	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
12:49:34.713 	at java.base/java.io.BufferedReader.readLine(BufferedReader.java:326)
12:49:34.713 	at java.base/java.io.BufferedReader.readLine(BufferedReader.java:392)
12:49:34.713 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild(OpenshiftProcessor.java:453)
12:49:34.714 	... 24 more
```

I think this is happening after Dekorate was upgrated to 2.x by [this change](https://github.com/quarkusio/quarkus/commit/e7acfd4a687e75aaab37f6c0778b678a0ae2a629).

## To Reproduce

Steps to reproduce the behavior:
1. `mvn io.quarkus:quarkus-maven-plugin:999-SNAPSHOT:create     -DprojectGroupId=org.acme     -DprojectArtifactId=openshift-quickstart     -DclassName="org.acme.rest.GreetingResource"     -Dpath="/greeting"  -Dextensions=openshift`
2. cd `openshift-quickstart`
3. Deploy it using `mvn package -Dquarkus.kubernetes.deploy=true -Dquarkus.openshift.expose=true`

Note that this is not happening always, but when happens this is really annoying...

I'm noticing this same issue frequently when building a quarkus app using the openshift extension.

```
Caused by: java.io.IOException: Pipe closed
    at java.io.PipedInputStream.read (PipedInputStream.java:307)
    at java.io.PipedInputStream.read (PipedInputStream.java:377)
    at sun.nio.cs.StreamDecoder.readBytes (StreamDecoder.java:284)
    at sun.nio.cs.StreamDecoder.implRead (StreamDecoder.java:326)
    at sun.nio.cs.StreamDecoder.read (StreamDecoder.java:178)
    at java.io.InputStreamReader.read (InputStreamReader.java:181)
    at java.io.BufferedReader.fill (BufferedReader.java:161)
    at java.io.BufferedReader.readLine (BufferedReader.java:326)
    at java.io.BufferedReader.readLine (BufferedReader.java:392)
    at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild (OpenshiftProcessor.java:444)
```

Env:
Quarkus 2.1.0.Final
Openshift 4.7

I'm facing this issue as well. 

[ERROR] Failed to execute goal io.quarkus.platform:quarkus-maven-plugin:2.2.1.Final:build (default) on project openshift-quickstart: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
[ERROR] 	[error]: Build step io.quarkus.container.image.openshift.deployment.OpenshiftProcessor#openshiftBuildFromJar threw an exception: java.lang.RuntimeException: Execution of openshift build failed. See build output for more details
[ERROR] 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftException(OpenshiftProcessor.java:502)
[ERROR] 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild(OpenshiftProcessor.java:448)
[ERROR] 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.lambda$openshiftBuild$10(OpenshiftProcessor.java:401)
[ERROR] 	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
[ERROR] 	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
[ERROR] 	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
[ERROR] 	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655)
[ERROR] 	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
[ERROR] 	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
[ERROR] 	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
[ERROR] 	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
[ERROR] 	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
[ERROR] 	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
[ERROR] 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild(OpenshiftProcessor.java:401)
[ERROR] 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.createContainerImage(OpenshiftProcessor.java:345)
[ERROR] 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuildFromJar(OpenshiftProcessor.java:252)
[ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[ERROR] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[ERROR] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[ERROR] 	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:820)
[ERROR] 	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
[ERROR] 	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
[ERROR] 	at java.base/java.lang.Thread.run(Thread.java:829)
[ERROR] 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
[ERROR] Caused by: java.io.IOException: Pipe closed
[ERROR] 	at java.base/java.io.PipedInputStream.read(PipedInputStream.java:307)
[ERROR] 	at java.base/java.io.PipedInputStream.read(PipedInputStream.java:377)
[ERROR] 	at java.base/sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
[ERROR] 	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
[ERROR] 	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
[ERROR] 	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:181)
[ERROR] 	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
[ERROR] 	at java.base/java.io.BufferedReader.readLine(BufferedReader.java:326)
[ERROR] 	at java.base/java.io.BufferedReader.readLine(BufferedReader.java:392)
[ERROR] 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild(OpenshiftProcessor.java:444)
[ERROR] 	... 25 more

Another example:

```
03:08:05.131 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] Generating dockerfile with builder image registry.access.redhat.com/ubi8/openjdk-11@sha256:76d357448c817d62aee4dcbbec20ac4c64211448158f0256b4d7b05a6646f706
03:08:05.136 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] Adding transient rw bind mount for /run/secrets/rhsm
03:08:05.136 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] STEP 1: FROM registry.access.redhat.com/ubi8/openjdk-11@sha256:76d357448c817d62aee4dcbbec20ac4c64211448158f0256b4d7b05a6646f706
03:08:05.137 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] STEP 2: LABEL "io.openshift.build.image"="registry.access.redhat.com/ubi8/openjdk-11@sha256:76d357448c817d62aee4dcbbec20ac4c64211448158f0256b4d7b05a6646f706"       "io.openshift.build.source-location"="/tmp/build/inputs"       "io.openshift.s2i.destination"="/tmp"
03:08:05.137 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] STEP 3: ENV OPENSHIFT_BUILD_NAME="app-1"     OPENSHIFT_BUILD_NAMESPACE="ts-giyagwaanu"
03:08:05.137 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] STEP 4: USER root
03:08:05.137 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] STEP 5: COPY upload/src /tmp/src
03:08:05.138 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] STEP 6: RUN chown -R 185:0 /tmp/src
03:08:05.138 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] STEP 7: USER 185
03:08:05.138 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] STEP 8: RUN /usr/local/s2i/assemble
03:08:05.138 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] INFO S2I source build with plain binaries detected
03:08:05.138 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] INFO Copying binaries from /tmp/src to /deployments ...
03:08:05.138 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] quarkus-app-dependencies.txt
03:08:05.138 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] quarkus-run.jar
03:08:05.138 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] app/
03:08:05.139 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] app/examples-pingpong-0.0.10-SNAPSHOT.jar
03:08:05.139 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/
03:08:05.139 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/boot/
03:08:05.139 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/boot/io.quarkus.quarkus-bootstrap-runner-2.2.3.Final.jar
03:08:05.139 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/boot/io.quarkus.quarkus-development-mode-spi-2.2.3.Final.jar
03:08:05.140 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/boot/io.smallrye.common.smallrye-common-io-1.6.0.jar
03:08:05.140 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/boot/org.graalvm.sdk.graal-sdk-21.2.0.jar
03:08:05.140 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/boot/org.jboss.logging.jboss-logging-3.4.2.Final.jar
03:08:05.140 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/boot/org.jboss.logmanager.jboss-logmanager-embedded-1.0.9.jar
03:08:05.141 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/boot/org.wildfly.common.wildfly-common-1.5.4.Final-format-001.jar
03:08:05.141 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/
03:08:05.148 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/com.fasterxml.jackson.core.jackson-core-2.12.4.jar
03:08:05.149 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/com.ibm.async.asyncutil-0.1.0.jar
03:08:05.149 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/com.sun.activation.jakarta.activation-1.2.1.jar
03:08:05.149 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.netty.netty-buffer-4.1.67.Final.jar
03:08:05.149 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.netty.netty-codec-4.1.67.Final.jar
03:08:05.149 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.netty.netty-codec-dns-4.1.67.Final.jar
03:08:05.149 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.netty.netty-codec-haproxy-4.1.67.Final.jar
03:08:05.149 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.netty.netty-codec-http-4.1.67.Final.jar
03:08:05.150 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.netty.netty-codec-http2-4.1.67.Final.jar
03:08:05.150 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.netty.netty-codec-socks-4.1.67.Final.jar
03:08:05.150 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.netty.netty-common-4.1.67.Final.jar
03:08:05.150 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.netty.netty-handler-4.1.67.Final.jar
03:08:05.150 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.netty.netty-handler-proxy-4.1.67.Final.jar
03:08:05.150 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.netty.netty-resolver-4.1.67.Final.jar
03:08:05.150 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.netty.netty-resolver-dns-4.1.67.Final.jar
03:08:05.150 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.netty.netty-transport-4.1.67.Final.jar
03:08:05.150 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.arc.arc-2.2.3.Final.jar
03:08:05.150 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.quarkus-arc-2.2.3.Final.jar
03:08:05.150 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.quarkus-container-image-2.2.3.Final.jar
03:08:05.150 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.quarkus-container-image-openshift-2.2.3.Final.jar
03:08:05.150 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.quarkus-core-2.2.3.Final.jar
03:08:05.150 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.quarkus-kubernetes-2.2.3.Final.jar
03:08:05.150 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.quarkus-kubernetes-client-internal-2.2.3.Final.jar
03:08:05.151 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.quarkus-mutiny-2.2.3.Final.jar
03:08:05.151 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.quarkus-netty-2.2.3.Final.jar
03:08:05.151 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.quarkus-openshift-2.2.3.Final.jar
03:08:05.151 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.quarkus-resteasy-2.2.3.Final.jar
03:08:05.151 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.quarkus-resteasy-common-2.2.3.Final.jar
03:08:05.151 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.quarkus-resteasy-server-common-2.2.3.Final.jar
03:08:05.151 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.quarkus-security-runtime-spi-2.2.3.Final.jar
03:08:05.151 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.quarkus-smallrye-context-propagation-2.2.3.Final.jar
03:08:05.151 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.quarkus-vertx-core-2.2.3.Final.jar
03:08:05.151 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.quarkus-vertx-http-2.2.3.Final.jar
03:08:05.151 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.quarkus-vertx-http-dev-console-runtime-spi-2.2.3.Final.jar
03:08:05.151 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.quarkus.security.quarkus-security-1.1.4.Final.jar
03:08:05.151 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.smallrye.common.smallrye-common-annotation-1.6.0.jar
03:08:05.151 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.smallrye.common.smallrye-common-classloader-1.6.0.jar
03:08:05.151 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.smallrye.common.smallrye-common-constraint-1.6.0.jar
03:08:05.152 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.smallrye.common.smallrye-common-expression-1.6.0.jar
03:08:05.152 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.smallrye.common.smallrye-common-function-1.6.0.jar
03:08:05.152 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.smallrye.config.smallrye-config-2.4.4.jar
03:08:05.152 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.smallrye.config.smallrye-config-common-2.4.4.jar
03:08:05.152 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.smallrye.config.smallrye-config-core-2.4.4.jar
03:08:05.152 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.smallrye.reactive.mutiny-1.0.0.jar
03:08:05.152 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.smallrye.reactive.mutiny-smallrye-context-propagation-1.0.0.jar
03:08:05.157 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.smallrye.smallrye-context-propagation-1.2.0.jar
03:08:05.157 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.smallrye.smallrye-context-propagation-api-1.2.0.jar
03:08:05.157 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.smallrye.smallrye-context-propagation-storage-1.2.0.jar
03:08:05.157 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.vertx.vertx-auth-common-4.1.3.jar
03:08:05.157 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.vertx.vertx-bridge-common-4.1.3.jar
03:08:05.157 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.vertx.vertx-core-4.1.3.jar
03:08:05.157 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.vertx.vertx-web-4.1.3.jar
03:08:05.158 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/io.vertx.vertx-web-common-4.1.3.jar
03:08:05.158 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/jakarta.annotation.jakarta.annotation-api-1.3.5.jar
03:08:05.158 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/jakarta.el.jakarta.el-api-3.0.3.jar
03:08:05.158 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/jakarta.enterprise.jakarta.enterprise.cdi-api-2.0.2.jar
03:08:05.158 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/jakarta.inject.jakarta.inject-api-1.0.jar
03:08:05.158 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/jakarta.interceptor.jakarta.interceptor-api-1.2.5.jar
03:08:05.158 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/jakarta.transaction.jakarta.transaction-api-1.3.3.jar
03:08:05.158 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/jakarta.validation.jakarta.validation-api-2.0.2.jar
03:08:05.158 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/org.eclipse.microprofile.config.microprofile-config-api-2.0.jar
03:08:05.158 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/org.eclipse.microprofile.context-propagation.microprofile-context-propagation-api-1.2.jar
03:08:05.159 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/org.jboss.logging.jboss-logging-annotations-2.2.1.Final.jar
03:08:05.159 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/org.jboss.resteasy.resteasy-core-4.7.0.Final.jar
03:08:05.159 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/org.jboss.resteasy.resteasy-core-spi-4.7.0.Final.jar
03:08:05.159 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/org.jboss.slf4j.slf4j-jboss-logmanager-1.1.0.Final.jar
03:08:05.159 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/org.jboss.spec.javax.ws.rs.jboss-jaxrs-api_2.1_spec-2.0.1.Final.jar
03:08:05.159 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/org.jboss.spec.javax.xml.bind.jboss-jaxb-api_2.3_spec-2.0.0.Final.jar
03:08:05.159 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/org.jboss.threads.jboss-threads-3.4.2.Final.jar
03:08:05.159 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/org.reactivestreams.reactive-streams-1.0.3.jar
03:08:05.159 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] lib/main/org.slf4j.slf4j-api-1.7.30.jar
03:08:05.159 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] quarkus/
03:08:05.159 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] quarkus/generated-bytecode.jar
03:08:05.159 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] quarkus/quarkus-application.dat
03:08:05.160 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] quarkus/transformed-bytecode.jar
03:08:05.160 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] INFO Cleaning up source directory (/tmp/src)
03:08:05.160 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] STEP 9: CMD /usr/local/s2i/run
03:08:05.160 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] STEP 10: COMMIT temp.builder.openshift.io/ts-giyagwaanu/app-1:342d5052
03:08:05.160 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] Getting image source signatures
03:08:05.160 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] Copying blob sha256:038780c13293b41959903d3509be954867f5a58192e7a0ce3f8978fed2566304
03:08:05.161 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] Copying blob sha256:05e0da429dc261e9197e56b8ca8e24a283cc5523127b07f6127d792b3f6c811e
03:08:05.161 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] Copying blob sha256:221e678a309f83be92eabdc9f1239aa4811eff9b782053d4f5053964210c428e
03:08:05.161 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] Copying blob sha256:38671b36593892ea76db64dd2f464a9419ae2dff37d4c586134eee0f1d8c0d28
03:08:05.161 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] Copying config sha256:067cfc1df92a6f99f74f7ff2adf062c338fa529f02aa067cf6de76a6680df16d
03:08:05.162 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] Writing manifest to image destination
03:08:05.162 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] Storing signatures
03:08:05.162 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] --> 067cfc1df92
03:08:05.162 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] 067cfc1df92a6f99f74f7ff2adf062c338fa529f02aa067cf6de76a6680df16d
03:08:05.162 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] 
03:08:05.163 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] Pushing image image-registry.openshift-image-registry.svc:5000/ts-giyagwaanu/app:0.0.10-SNAPSHOT ...
03:08:05.163 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] Getting image source signatures
03:08:05.164 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] Copying blob sha256:38671b36593892ea76db64dd2f464a9419ae2dff37d4c586134eee0f1
03:08:05.189 INFO  mvn: Exception in thread "pool-7-thread-1" io.fabric8.kubernetes.client.KubernetesClientException: Operation: [list]  for kind: [Build]  with name: [app-1]  in namespace: [ts-giyagwaanu]  failed.
03:08:05.189 INFO  mvn: 	at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:64)
03:08:05.190 INFO  mvn: 	at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:72)
03:08:05.190 INFO  mvn: 	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.listRequestHelper(BaseOperation.java:144)
03:08:05.190 INFO  mvn: 	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.list(BaseOperation.java:524)
03:08:05.190 INFO  mvn: 	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.list(BaseOperation.java:513)
03:08:05.190 INFO  mvn: 	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.list(BaseOperation.java:88)
03:08:05.190 INFO  mvn: 	at io.fabric8.kubernetes.client.informers.cache.Reflector.getList(Reflector.java:53)
03:08:05.190 INFO  mvn: 	at io.fabric8.kubernetes.client.informers.cache.Reflector.listSyncAndWatch(Reflector.java:77)
03:08:05.190 INFO  mvn: 	at io.fabric8.kubernetes.client.informers.impl.DefaultSharedIndexInformer.run(DefaultSharedIndexInformer.java:146)
03:08:05.190 INFO  mvn: 	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.informOnCondition(BaseOperation.java:1007)
03:08:05.191 INFO  mvn: 	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.waitUntilCondition(BaseOperation.java:946)
03:08:05.192 INFO  mvn: 	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.waitUntilCondition(BaseOperation.java:88)
03:08:05.192 INFO  mvn: 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.lambda$watchBuild$12(OpenshiftProcessor.java:468)
03:08:05.192 INFO  mvn: 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
03:08:05.192 INFO  mvn: 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
03:08:05.192 INFO  mvn: 	at java.base/java.lang.Thread.run(Thread.java:834)
03:08:05.192 INFO  mvn: Caused by: javax.net.ssl.SSLException: Socket closed
03:08:05.192 INFO  mvn: 	at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:127)
03:08:05.193 INFO  mvn: 	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:369)
03:08:05.193 INFO  mvn: 	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:312)
03:08:05.193 INFO  mvn: 	at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:307)
03:08:05.193 INFO  mvn: 	at java.base/sun.security.ssl.SSLTransport.decode(SSLTransport.java:144)
03:08:05.193 INFO  mvn: 	at java.base/sun.security.ssl.SSLSocketImpl.decode(SSLSocketImpl.java:1369)
03:08:05.193 INFO  mvn: 	at java.base/sun.security.ssl.SSLSocketImpl.readHandshakeRecord(SSLSocketImpl.java:1278)
03:08:05.193 INFO  mvn: 	at java.base/sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:401)
03:08:05.193 INFO  mvn: 	at java.base/sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:373)
03:08:05.193 INFO  mvn: 	at okhttp3.internal.connection.RealConnection.connectTls(RealConnection.java:336)
03:08:05.194 INFO  mvn: 	at okhttp3.internal.connection.RealConnection.establishProtocol(RealConnection.java:300)
03:08:05.194 INFO  mvn: 	at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:185)
03:08:05.194 INFO  mvn: 	at okhttp3.internal.connection.ExchangeFinder.findConnection(ExchangeFinder.java:224)
03:08:05.194 INFO  mvn: 	at okhttp3.internal.connection.ExchangeFinder.findHealthyConnection(ExchangeFinder.java:108)
03:08:05.194 INFO  mvn: 	at okhttp3.internal.connection.ExchangeFinder.find(ExchangeFinder.java:88)
03:08:05.194 INFO  mvn: 	at okhttp3.internal.connection.Transmitter.newExchange(Transmitter.java:169)
03:08:05.194 INFO  mvn: 	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:41)
03:08:05.195 INFO  mvn: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
03:08:05.195 INFO  mvn: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
03:08:05.195 INFO  mvn: 	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:94)
03:08:05.195 INFO  mvn: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
03:08:05.195 INFO  mvn: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
03:08:05.195 INFO  mvn: 	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
03:08:05.195 INFO  mvn: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
03:08:05.195 INFO  mvn: 	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:88)
03:08:05.196 INFO  mvn: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
03:08:05.196 INFO  mvn: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
03:08:05.196 INFO  mvn: 	at io.fabric8.kubernetes.client.utils.BackwardsCompatibilityInterceptor.intercept(BackwardsCompatibilityInterceptor.java:133)
03:08:05.196 INFO  mvn: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
03:08:05.196 INFO  mvn: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
03:08:05.196 INFO  mvn: 	at io.fabric8.kubernetes.client.utils.ImpersonatorInterceptor.intercept(ImpersonatorInterceptor.java:68)
03:08:05.196 INFO  mvn: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
03:08:05.197 INFO  mvn: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
03:08:05.197 INFO  mvn: 	at io.fabric8.openshift.client.internal.OpenShiftOAuthInterceptor.intercept(OpenShiftOAuthInterceptor.java:92)
03:08:05.197 INFO  mvn: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)
03:08:05.197 INFO  mvn: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)
03:08:05.197 INFO  mvn: 	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:229)
03:08:05.198 INFO  mvn: 	at okhttp3.RealCall.execute(RealCall.java:81)
03:08:05.198 INFO  mvn: 	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.retryWithExponentialBackoff(OperationSupport.java:590)
03:08:05.198 INFO  mvn: 	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:563)
03:08:05.198 INFO  mvn: 	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:526)
03:08:05.198 INFO  mvn: 	at io.fabric8.kubernetes.client.dsl.base.OperationSupport.handleResponse(OperationSupport.java:509)
03:08:05.198 INFO  mvn: 	at io.fabric8.kubernetes.client.dsl.base.BaseOperation.listRequestHelper(BaseOperation.java:137)
03:08:05.198 INFO  mvn: 	... 13 more
03:08:05.198 INFO  mvn: 	Suppressed: java.net.SocketException: Socket closed
03:08:05.198 INFO  mvn: 		at java.base/java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:113)
03:08:05.199 INFO  mvn: 		at java.base/java.net.SocketOutputStream.write(SocketOutputStream.java:150)
03:08:05.199 INFO  mvn: 		at java.base/sun.security.ssl.SSLSocketOutputRecord.encodeAlert(SSLSocketOutputRecord.java:83)
03:08:05.199 INFO  mvn: 		at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:400)
03:08:05.199 INFO  mvn: 		... 54 more
03:08:05.199 INFO  mvn: Caused by: java.net.SocketException: Socket closed
03:08:05.199 INFO  mvn: 	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:183)
03:08:05.200 INFO  mvn: 	at java.base/java.net.SocketInputStream.read(SocketInputStream.java:140)
03:08:05.200 INFO  mvn: 	at java.base/sun.security.ssl.SSLSocketInputRecord.read(SSLSocketInputRecord.java:478)
03:08:05.200 INFO  mvn: 	at java.base/sun.security.ssl.SSLSocketInputRecord.readHeader(SSLSocketInputRecord.java:472)
03:08:05.200 INFO  mvn: 	at java.base/sun.security.ssl.SSLSocketInputRecord.decode(SSLSocketInputRecord.java:160)
03:08:05.200 INFO  mvn: 	at java.base/sun.security.ssl.SSLTransport.decode(SSLTransport.java:110)
03:08:05.200 INFO  mvn: 	... 51 more
```

And another example:

```
04:14:49.590 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] Cleanup: popt;1.16-14.el8;x86_64;installed
04:14:49.591 INFO  mvn: [INFO] [io.quarkus.container.image.openshift.deployment.OpenshiftProcessor] Cleanup: brotli;1.0.6-2.el8;x86_64;installed
04:14:49.620 INFO  mvn: [INFO] ------------------------------------------------------------------------
04:14:49.620 INFO  mvn: [INFO] BUILD FAILURE
04:14:49.620 INFO  mvn: [INFO] ------------------------------------------------------------------------
04:14:49.620 INFO  mvn: [INFO] Total time:  01:08 min
04:14:49.620 INFO  mvn: [INFO] Finished at: 2021-09-14T04:14:49Z
04:14:49.620 INFO  mvn: [INFO] ------------------------------------------------------------------------
04:14:49.621 INFO  mvn: [ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:2.2.3.Final:build (build) on project http-minimum: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
04:14:49.621 INFO  mvn: [ERROR] 	[error]: Build step io.quarkus.container.image.openshift.deployment.OpenshiftProcessor#openshiftBuildFromJar threw an exception: java.lang.RuntimeException: Execution of openshift build failed. See build output for more details
04:14:49.621 INFO  mvn: [ERROR] 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftException(OpenshiftProcessor.java:502)
04:14:49.621 INFO  mvn: [ERROR] 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild(OpenshiftProcessor.java:448)
04:14:49.621 INFO  mvn: [ERROR] 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.lambda$openshiftBuild$10(OpenshiftProcessor.java:401)
04:14:49.621 INFO  mvn: [ERROR] 	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
04:14:49.621 INFO  mvn: [ERROR] 	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
04:14:49.621 INFO  mvn: [ERROR] 	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
04:14:49.621 INFO  mvn: [ERROR] 	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655)
04:14:49.621 INFO  mvn: [ERROR] 	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
04:14:49.621 INFO  mvn: [ERROR] 	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
04:14:49.621 INFO  mvn: [ERROR] 	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
04:14:49.621 INFO  mvn: [ERROR] 	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
04:14:49.622 INFO  mvn: [ERROR] 	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
04:14:49.622 INFO  mvn: [ERROR] 	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
04:14:49.622 INFO  mvn: [ERROR] 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild(OpenshiftProcessor.java:401)
04:14:49.622 INFO  mvn: [ERROR] 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.createContainerImage(OpenshiftProcessor.java:345)
04:14:49.622 INFO  mvn: [ERROR] 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuildFromJar(OpenshiftProcessor.java:252)
04:14:49.622 INFO  mvn: [ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
04:14:49.622 INFO  mvn: [ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
04:14:49.622 INFO  mvn: [ERROR] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
04:14:49.622 INFO  mvn: [ERROR] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
04:14:49.622 INFO  mvn: [ERROR] 	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:820)
04:14:49.623 INFO  mvn: [ERROR] 	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
04:14:49.623 INFO  mvn: [ERROR] 	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
04:14:49.623 INFO  mvn: [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
04:14:49.625 INFO  mvn: [ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
04:14:49.625 INFO  mvn: [ERROR] 	at java.base/java.lang.Thread.run(Thread.java:834)
04:14:49.625 INFO  mvn: [ERROR] 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
04:14:49.625 INFO  mvn: [ERROR] Caused by: java.io.IOException: Pipe closed
04:14:49.625 INFO  mvn: [ERROR] 	at java.base/java.io.PipedInputStream.read(PipedInputStream.java:307)
04:14:49.626 INFO  mvn: [ERROR] 	at java.base/java.io.PipedInputStream.read(PipedInputStream.java:377)
04:14:49.626 INFO  mvn: [ERROR] 	at java.base/sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
04:14:49.626 INFO  mvn: [ERROR] 	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
04:14:49.626 INFO  mvn: [ERROR] 	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
04:14:49.626 INFO  mvn: [ERROR] 	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:181)
04:14:49.626 INFO  mvn: [ERROR] 	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
04:14:49.626 INFO  mvn: [ERROR] 	at java.base/java.io.BufferedReader.readLine(BufferedReader.java:326)
04:14:49.627 INFO  mvn: [ERROR] 	at java.base/java.io.BufferedReader.readLine(BufferedReader.java:392)
04:14:49.627 INFO  mvn: [ERROR] 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild(OpenshiftProcessor.java:444)
04:14:49.627 INFO  mvn: [ERROR] 	... 25 more
04:14:49.627 INFO  mvn: [ERROR] -> [Help 1]
04:14:49.627 INFO  mvn: org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.quarkus:quarkus-maven-plugin:2.2.3.Final:build (build) on project http-minimum: Failed to build quarkus application
04:14:49.627 INFO  mvn:     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
04:14:49.628 INFO  mvn:     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
04:14:49.628 INFO  mvn:     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
04:14:49.628 INFO  mvn:     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
04:14:49.628 INFO  mvn:     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
04:14:49.628 INFO  mvn:     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
04:14:49.628 INFO  mvn:     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
04:14:49.629 INFO  mvn:     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
04:14:49.629 INFO  mvn:     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
04:14:49.629 INFO  mvn:     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
04:14:49.629 INFO  mvn:     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
04:14:49.629 INFO  mvn:     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
04:14:49.629 INFO  mvn:     at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
04:14:49.629 INFO  mvn:     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
04:14:49.629 INFO  mvn:     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
04:14:49.629 INFO  mvn:     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
04:14:49.629 INFO  mvn:     at java.lang.reflect.Method.invoke (Method.java:566)
04:14:49.630 INFO  mvn:     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
04:14:49.630 INFO  mvn:     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
04:14:49.630 INFO  mvn:     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
04:14:49.630 INFO  mvn:     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
04:14:49.630 INFO  mvn: Caused by: org.apache.maven.plugin.MojoExecutionException: Failed to build quarkus application
04:14:49.630 INFO  mvn:     at io.quarkus.maven.BuildMojo.doExecute (BuildMojo.java:162)
04:14:49.630 INFO  mvn:     at io.quarkus.maven.QuarkusBootstrapMojo.execute (QuarkusBootstrapMojo.java:125)
04:14:49.630 INFO  mvn:     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
04:14:49.631 INFO  mvn:     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
04:14:49.631 INFO  mvn:     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
04:14:49.631 INFO  mvn:     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
04:14:49.631 INFO  mvn:     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
04:14:49.631 INFO  mvn:     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
04:14:49.631 INFO  mvn:     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
04:14:49.631 INFO  mvn:     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
04:14:49.631 INFO  mvn:     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
04:14:49.631 INFO  mvn:     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
04:14:49.635 INFO  mvn:     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
04:14:49.635 INFO  mvn:     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
04:14:49.635 INFO  mvn:     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
04:14:49.636 INFO  mvn:     at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
04:14:49.636 INFO  mvn:     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
04:14:49.636 INFO  mvn:     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
04:14:49.636 INFO  mvn:     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
04:14:49.636 INFO  mvn:     at java.lang.reflect.Method.invoke (Method.java:566)
04:14:49.636 INFO  mvn:     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
04:14:49.636 INFO  mvn:     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
04:14:49.637 INFO  mvn:     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
04:14:49.637 INFO  mvn:     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
04:14:49.637 INFO  mvn: Caused by: java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
04:14:49.637 INFO  mvn: 	[error]: Build step io.quarkus.container.image.openshift.deployment.OpenshiftProcessor#openshiftBuildFromJar threw an exception: java.lang.RuntimeException: Execution of openshift build failed. See build output for more details
04:14:49.637 INFO  mvn: 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftException(OpenshiftProcessor.java:502)
04:14:49.637 INFO  mvn: 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild(OpenshiftProcessor.java:448)
04:14:49.637 INFO  mvn: 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.lambda$openshiftBuild$10(OpenshiftProcessor.java:401)
04:14:49.638 INFO  mvn: 	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
04:14:49.638 INFO  mvn: 	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
04:14:49.638 INFO  mvn: 	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
04:14:49.638 INFO  mvn: 	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655)
04:14:49.638 INFO  mvn: 	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
04:14:49.638 INFO  mvn: 	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
04:14:49.639 INFO  mvn: 	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
04:14:49.639 INFO  mvn: 	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
04:14:49.639 INFO  mvn: 	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
04:14:49.639 INFO  mvn: 	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
04:14:49.639 INFO  mvn: 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild(OpenshiftProcessor.java:401)
04:14:49.639 INFO  mvn: 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.createContainerImage(OpenshiftProcessor.java:345)
04:14:49.639 INFO  mvn: 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuildFromJar(OpenshiftProcessor.java:252)
04:14:49.640 INFO  mvn: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
04:14:49.640 INFO  mvn: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
04:14:49.640 INFO  mvn: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
04:14:49.640 INFO  mvn: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
04:14:49.640 INFO  mvn: 	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:820)
04:14:49.640 INFO  mvn: 	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
04:14:49.640 INFO  mvn: 	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
04:14:49.641 INFO  mvn: 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
04:14:49.657 INFO  mvn: 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
04:14:49.657 INFO  mvn: 	at java.base/java.lang.Thread.run(Thread.java:834)
04:14:49.657 INFO  mvn: 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
04:14:49.657 INFO  mvn: Caused by: java.io.IOException: Pipe closed
04:14:49.657 INFO  mvn: 	at java.base/java.io.PipedInputStream.read(PipedInputStream.java:307)
04:14:49.657 INFO  mvn: 	at java.base/java.io.PipedInputStream.read(PipedInputStream.java:377)
04:14:49.657 INFO  mvn: 	at java.base/sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
04:14:49.657 INFO  mvn: 	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
04:14:49.657 INFO  mvn: 	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
04:14:49.657 INFO  mvn: 	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:181)
04:14:49.657 INFO  mvn: 	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
04:14:49.657 INFO  mvn: 	at java.base/java.io.BufferedReader.readLine(BufferedReader.java:326)
04:14:49.658 INFO  mvn: 	at java.base/java.io.BufferedReader.readLine(BufferedReader.java:392)
04:14:49.658 INFO  mvn: 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild(OpenshiftProcessor.java:444)
04:14:49.658 INFO  mvn: 	... 25 more
04:14:49.658 INFO  mvn: 
04:14:49.658 INFO  mvn:     at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment (AugmentActionImpl.java:415)
04:14:49.658 INFO  mvn:     at io.quarkus.runner.bootstrap.AugmentActionImpl.createProductionApplication (AugmentActionImpl.java:202)
04:14:49.658 INFO  mvn:     at io.quarkus.maven.BuildMojo.doExecute (BuildMojo.java:132)
04:14:49.658 INFO  mvn:     at io.quarkus.maven.QuarkusBootstrapMojo.execute (QuarkusBootstrapMojo.java:125)
04:14:49.658 INFO  mvn:     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
04:14:49.658 INFO  mvn:     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
04:14:49.658 INFO  mvn:     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
04:14:49.659 INFO  mvn:     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
04:14:49.659 INFO  mvn:     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
04:14:49.659 INFO  mvn:     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
04:14:49.659 INFO  mvn:     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
04:14:49.659 INFO  mvn:     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
04:14:49.659 INFO  mvn:     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
04:14:49.659 INFO  mvn:     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
04:14:49.659 INFO  mvn:     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
04:14:49.660 INFO  mvn:     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
04:14:49.660 INFO  mvn:     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
04:14:49.660 INFO  mvn:     at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
04:14:49.660 INFO  mvn:     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
04:14:49.660 INFO  mvn:     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
04:14:49.660 INFO  mvn:     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
04:14:49.661 INFO  mvn:     at java.lang.reflect.Method.invoke (Method.java:566)
04:14:49.661 INFO  mvn:     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
04:14:49.661 INFO  mvn:     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
04:14:49.661 INFO  mvn:     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
04:14:49.661 INFO  mvn:     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
04:14:49.662 INFO  mvn: Caused by: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
04:14:49.662 INFO  mvn: 	[error]: Build step io.quarkus.container.image.openshift.deployment.OpenshiftProcessor#openshiftBuildFromJar threw an exception: java.lang.RuntimeException: Execution of openshift build failed. See build output for more details
04:14:49.662 INFO  mvn: 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftException(OpenshiftProcessor.java:502)
04:14:49.662 INFO  mvn: 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild(OpenshiftProcessor.java:448)
04:14:49.663 INFO  mvn: 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.lambda$openshiftBuild$10(OpenshiftProcessor.java:401)
04:14:49.663 INFO  mvn: 	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
04:14:49.663 INFO  mvn: 	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
04:14:49.663 INFO  mvn: 	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
04:14:49.663 INFO  mvn: 	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655)
04:14:49.664 INFO  mvn: 	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
04:14:49.664 INFO  mvn: 	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
04:14:49.664 INFO  mvn: 	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
04:14:49.675 INFO  mvn: 	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
04:14:49.676 INFO  mvn: 	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
04:14:49.676 INFO  mvn: 	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
04:14:49.676 INFO  mvn: 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild(OpenshiftProcessor.java:401)
04:14:49.676 INFO  mvn: 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.createContainerImage(OpenshiftProcessor.java:345)
04:14:49.676 INFO  mvn: 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuildFromJar(OpenshiftProcessor.java:252)
04:14:49.676 INFO  mvn: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
04:14:49.676 INFO  mvn: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
04:14:49.676 INFO  mvn: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
04:14:49.676 INFO  mvn: 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
04:14:49.677 INFO  mvn: 	at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:820)
04:14:49.677 INFO  mvn: 	at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
04:14:49.677 INFO  mvn: 	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
04:14:49.677 INFO  mvn: 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
04:14:49.677 INFO  mvn: 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
04:14:49.678 INFO  mvn: 	at java.base/java.lang.Thread.run(Thread.java:834)
04:14:49.678 INFO  mvn: 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
04:14:49.678 INFO  mvn: Caused by: java.io.IOException: Pipe closed
04:14:49.678 INFO  mvn: 	at java.base/java.io.PipedInputStream.read(PipedInputStream.java:307)
04:14:49.678 INFO  mvn: 	at java.base/java.io.PipedInputStream.read(PipedInputStream.java:377)
04:14:49.678 INFO  mvn: 	at java.base/sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
04:14:49.678 INFO  mvn: 	at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
04:14:49.678 INFO  mvn: 	at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
04:14:49.678 INFO  mvn: 	at java.base/java.io.InputStreamReader.read(InputStreamReader.java:181)
04:14:49.678 INFO  mvn: 	at java.base/java.io.BufferedReader.fill(BufferedReader.java:161)
04:14:49.679 INFO  mvn: 	at java.base/java.io.BufferedReader.readLine(BufferedReader.java:326)
04:14:49.679 INFO  mvn: 	at java.base/java.io.BufferedReader.readLine(BufferedReader.java:392)
04:14:49.679 INFO  mvn: 	at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild(OpenshiftProcessor.java:444)
04:14:49.679 INFO  mvn: 	... 25 more
04:14:49.679 INFO  mvn: 
04:14:49.679 INFO  mvn:     at io.quarkus.builder.Execution.run (Execution.java:116)
04:14:49.679 INFO  mvn:     at io.quarkus.builder.BuildExecutionBuilder.execute (BuildExecutionBuilder.java:79)
04:14:49.680 INFO  mvn:     at io.quarkus.deployment.QuarkusAugmentor.run (QuarkusAugmentor.java:151)
04:14:49.680 INFO  mvn:     at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment (AugmentActionImpl.java:413)
04:14:49.680 INFO  mvn:     at io.quarkus.runner.bootstrap.AugmentActionImpl.createProductionApplication (AugmentActionImpl.java:202)
04:14:49.680 INFO  mvn:     at io.quarkus.maven.BuildMojo.doExecute (BuildMojo.java:132)
04:14:49.680 INFO  mvn:     at io.quarkus.maven.QuarkusBootstrapMojo.execute (QuarkusBootstrapMojo.java:125)
04:14:49.680 INFO  mvn:     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
04:14:49.680 INFO  mvn:     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
04:14:49.680 INFO  mvn:     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
04:14:49.680 INFO  mvn:     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
04:14:49.680 INFO  mvn:     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
04:14:49.681 INFO  mvn:     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
04:14:49.681 INFO  mvn:     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
04:14:49.681 INFO  mvn:     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
04:14:49.681 INFO  mvn:     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
04:14:49.681 INFO  mvn:     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
04:14:49.681 INFO  mvn:     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
04:14:49.681 INFO  mvn:     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
04:14:49.682 INFO  mvn:     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
04:14:49.682 INFO  mvn:     at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
04:14:49.682 INFO  mvn:     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
04:14:49.682 INFO  mvn:     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
04:14:49.682 INFO  mvn:     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
04:14:49.682 INFO  mvn:     at java.lang.reflect.Method.invoke (Method.java:566)
04:14:49.682 INFO  mvn:     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
04:14:49.682 INFO  mvn:     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
04:14:49.682 INFO  mvn:     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
04:14:49.683 INFO  mvn:     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
04:14:49.683 INFO  mvn: Caused by: java.lang.RuntimeException: Execution of openshift build failed. See build output for more details
04:14:49.683 INFO  mvn:     at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftException (OpenshiftProcessor.java:502)
04:14:49.683 INFO  mvn:     at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild (OpenshiftProcessor.java:448)
04:14:49.683 INFO  mvn:     at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.lambda$openshiftBuild$10 (OpenshiftProcessor.java:401)
04:14:49.683 INFO  mvn:     at java.util.stream.ForEachOps$ForEachOp$OfRef.accept (ForEachOps.java:183)
04:14:49.694 INFO  mvn:     at java.util.stream.ReferencePipeline$3$1.accept (ReferencePipeline.java:195)
04:14:49.694 INFO  mvn:     at java.util.stream.ReferencePipeline$2$1.accept (ReferencePipeline.java:177)
04:14:49.694 INFO  mvn:     at java.util.ArrayList$ArrayListSpliterator.forEachRemaining (ArrayList.java:1655)
04:14:49.694 INFO  mvn:     at java.util.stream.AbstractPipeline.copyInto (AbstractPipeline.java:484)
04:14:49.695 INFO  mvn:     at java.util.stream.AbstractPipeline.wrapAndCopyInto (AbstractPipeline.java:474)
04:14:49.695 INFO  mvn:     at java.util.stream.ForEachOps$ForEachOp.evaluateSequential (ForEachOps.java:150)
04:14:49.695 INFO  mvn:     at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential (ForEachOps.java:173)
04:14:49.695 INFO  mvn:     at java.util.stream.AbstractPipeline.evaluate (AbstractPipeline.java:234)
04:14:49.695 INFO  mvn:     at java.util.stream.ReferencePipeline.forEach (ReferencePipeline.java:497)
04:14:49.696 INFO  mvn:     at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild (OpenshiftProcessor.java:401)
04:14:49.696 INFO  mvn:     at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.createContainerImage (OpenshiftProcessor.java:345)
04:14:49.696 INFO  mvn:     at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuildFromJar (OpenshiftProcessor.java:252)
04:14:49.696 INFO  mvn:     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
04:14:49.697 INFO  mvn:     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
04:14:49.697 INFO  mvn:     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
04:14:49.697 INFO  mvn:     at java.lang.reflect.Method.invoke (Method.java:566)
04:14:49.697 INFO  mvn:     at io.quarkus.deployment.ExtensionLoader$2.execute (ExtensionLoader.java:820)
04:14:49.697 INFO  mvn:     at io.quarkus.builder.BuildContext.run (BuildContext.java:277)
04:14:49.697 INFO  mvn:     at org.jboss.threads.ContextHandler$1.runWith (ContextHandler.java:18)
04:14:49.697 INFO  mvn:     at org.jboss.threads.EnhancedQueueExecutor$Task.run (EnhancedQueueExecutor.java:2449)
04:14:49.698 INFO  mvn:     at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run (EnhancedQueueExecutor.java:1478)
04:14:49.698 INFO  mvn:     at java.lang.Thread.run (Thread.java:834)
04:14:49.698 INFO  mvn:     at org.jboss.threads.JBossThread.run (JBossThread.java:501)
04:14:49.698 INFO  mvn: Caused by: java.io.IOException: Pipe closed
04:14:49.698 INFO  mvn:     at java.io.PipedInputStream.read (PipedInputStream.java:307)
04:14:49.698 INFO  mvn:     at java.io.PipedInputStream.read (PipedInputStream.java:377)
04:14:49.698 INFO  mvn:     at sun.nio.cs.StreamDecoder.readBytes (StreamDecoder.java:284)
04:14:49.698 INFO  mvn:     at sun.nio.cs.StreamDecoder.implRead (StreamDecoder.java:326)
04:14:49.699 INFO  mvn:     at sun.nio.cs.StreamDecoder.read (StreamDecoder.java:178)
04:14:49.699 INFO  mvn:     at java.io.InputStreamReader.read (InputStreamReader.java:181)
04:14:49.699 INFO  mvn:     at java.io.BufferedReader.fill (BufferedReader.java:161)
04:14:49.699 INFO  mvn:     at java.io.BufferedReader.readLine (BufferedReader.java:326)
04:14:49.699 INFO  mvn:     at java.io.BufferedReader.readLine (BufferedReader.java:392)
04:14:49.699 INFO  mvn:     at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild (OpenshiftProcessor.java:444)
04:14:49.700 INFO  mvn:     at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.lambda$openshiftBuild$10 (OpenshiftProcessor.java:401)
04:14:49.700 INFO  mvn:     at java.util.stream.ForEachOps$ForEachOp$OfRef.accept (ForEachOps.java:183)
04:14:49.700 INFO  mvn:     at java.util.stream.ReferencePipeline$3$1.accept (ReferencePipeline.java:195)
04:14:49.700 INFO  mvn:     at java.util.stream.ReferencePipeline$2$1.accept (ReferencePipeline.java:177)
04:14:49.701 INFO  mvn:     at java.util.ArrayList$ArrayListSpliterator.forEachRemaining (ArrayList.java:1655)
04:14:49.701 INFO  mvn:     at java.util.stream.AbstractPipeline.copyInto (AbstractPipeline.java:484)
04:14:49.701 INFO  mvn:     at java.util.stream.AbstractPipeline.wrapAndCopyInto (AbstractPipeline.java:474)
04:14:49.701 INFO  mvn:     at java.util.stream.ForEachOps$ForEachOp.evaluateSequential (ForEachOps.java:150)
04:14:49.701 INFO  mvn:     at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential (ForEachOps.java:173)
04:14:49.701 INFO  mvn:     at java.util.stream.AbstractPipeline.evaluate (AbstractPipeline.java:234)
04:14:49.701 INFO  mvn:     at java.util.stream.ReferencePipeline.forEach (ReferencePipeline.java:497)
04:14:49.702 INFO  mvn:     at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuild (OpenshiftProcessor.java:401)
04:14:49.702 INFO  mvn:     at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.createContainerImage (OpenshiftProcessor.java:345)
04:14:49.702 INFO  mvn:     at io.quarkus.container.image.openshift.deployment.OpenshiftProcessor.openshiftBuildFromJar (OpenshiftProcessor.java:252)
04:14:49.702 INFO  mvn:     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
04:14:49.702 INFO  mvn:     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
04:14:49.702 INFO  mvn:     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
04:14:49.703 INFO  mvn:     at java.lang.reflect.Method.invoke (Method.java:566)
04:14:49.703 INFO  mvn:     at io.quarkus.deployment.ExtensionLoader$2.execute (ExtensionLoader.java:820)
04:14:49.703 INFO  mvn:     at io.quarkus.builder.BuildContext.run (BuildContext.java:277)
04:14:49.703 INFO  mvn:     at org.jboss.threads.ContextHandler$1.runWith (ContextHandler.java:18)
04:14:49.703 INFO  mvn:     at org.jboss.threads.EnhancedQueueExecutor$Task.run (EnhancedQueueExecutor.java:2449)
04:14:49.703 INFO  mvn:     at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run (EnhancedQueueExecutor.java:1478)
04:14:49.704 INFO  mvn:     at java.lang.Thread.run (Thread.java:834)
04:14:49.704 INFO  mvn:     at org.jboss.threads.JBossThread.run (JBossThread.java:501)
04:14:49.704 INFO  mvn: [ERROR] 
04:14:49.704 INFO  mvn: [ERROR] Re-run Maven using the -X switch to enable full debug logging.
04:14:49.716 INFO  mvn: [ERROR] 
04:14:49.716 INFO  mvn: [ERROR] For more information about the errors and possible solutions, please read the following articles:
04:14:49.716 INFO  mvn: [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
``` 

The above two examples are based on 2.2.3.Final.

--- a/extensions/container-image/container-image-openshift/deployment/src/main/java/io/quarkus/container/image/openshift/deployment/OpenshiftProcessor.java
+++ b/extensions/container-image/container-image-openshift/deployment/src/main/java/io/quarkus/container/image/openshift/deployment/OpenshiftProcessor.java
@@ -6,10 +6,10 @@ import static io.quarkus.deployment.pkg.steps.JarResultBuildStep.DEFAULT_FAST_JA
 
 import java.io.BufferedReader;
 import java.io.ByteArrayInputStream;
-import java.io.Closeable;
 import java.io.File;
 import java.io.IOException;
 import java.io.InputStreamReader;
+import java.io.Reader;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.StandardCopyOption;
@@ -21,8 +21,6 @@ import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.Executor;
-import java.util.concurrent.Executors;
 import java.util.concurrent.TimeUnit;
 import java.util.function.Predicate;
 import java.util.stream.Collectors;
@@ -75,6 +73,7 @@ public class OpenshiftProcessor {
     private static final String BUILD_CONFIG_NAME = "openshift.io/build-config.name";
     private static final String RUNNING = "Running";
 
+    private static final int LOG_TAIL_SIZE = 10;
     private static final Logger LOG = Logger.getLogger(OpenshiftProcessor.class);
 
     @BuildStep
@@ -430,51 +429,39 @@ public class OpenshiftProcessor {
         }
 
         while (isNew(build) || isPending(build) || isRunning(build)) {
-            Build updated = client.builds().withName(build.getMetadata().getName()).get();
+            final String buildName = build.getMetadata().getName();
+            Build updated = client.builds().withName(buildName).get();
             if (updated == null) {
                 throw new IllegalStateException("Build:" + build.getMetadata().getName() + " is no longer present!");
             } else if (updated.getStatus() == null) {
                 throw new IllegalStateException("Build:" + build.getMetadata().getName() + " has no status!");
             } else if (isNew(updated) || isPending(updated) || isRunning(updated)) {
                 build = updated;
-                final String buildName = build.getMetadata().getName();
-                try (LogWatch w = client.builds().withName(build.getMetadata().getName()).withPrettyOutput().watchLog();
-                        BufferedReader reader = new BufferedReader(new InputStreamReader(w.getOutput()))) {
-                    watchBuild(client, openshiftConfig, buildName, w);
-                    for (String line = reader.readLine(); line != null; line = reader.readLine()) {
-                        LOG.info(line);
-                    }
+                try (LogWatch w = client.builds().withName(buildName).withPrettyOutput().watchLog();
+                        Reader reader = new InputStreamReader(w.getOutput())) {
+                    display(reader);
                 } catch (IOException e) {
-                    throw openshiftException(e);
+                    // This may happen if the LogWatch is closed while we are still reading.
+                    // We shouldn't let the build fail, so let's log a warning and display last few lines of the log
+                    LOG.warn("Log stream closed, redisplaying last " + LOG_TAIL_SIZE + " entries:");
+                    try {
+                        display(client.builds().withName(buildName).tailingLines(LOG_TAIL_SIZE).getLogReader());
+                    } catch (IOException ex) {
+                        // Let's ignore this.
+                    }
                 }
             } else if (isComplete(updated)) {
                 return;
             } else if (isCancelled(updated)) {
-                throw new IllegalStateException("Build:" + build.getMetadata().getName() + " cancelled!");
+                throw new IllegalStateException("Build:" + buildName + " cancelled!");
             } else if (isFailed(updated)) {
                 throw new IllegalStateException(
-                        "Build:" + build.getMetadata().getName() + " failed! " + updated.getStatus().getMessage());
+                        "Build:" + buildName + " failed! " + updated.getStatus().getMessage());
             } else if (isError(updated)) {
                 throw new IllegalStateException(
-                        "Build:" + build.getMetadata().getName() + " encountered error! " + updated.getStatus().getMessage());
+                        "Build:" + buildName + " encountered error! " + updated.getStatus().getMessage());
             }
         }
-    }
-
-    private static void watchBuild(OpenShiftClient client, OpenshiftConfig openshiftConfig, String buildName, Closeable watch) {
-        Executor executor = Executors.newSingleThreadExecutor();
-        executor.execute(() -> {
-            try {
-                client.builds().withName(buildName).waitUntilCondition(b -> !RUNNING.equalsIgnoreCase(b.getStatus().getPhase()),
-                        openshiftConfig.buildTimeout.toMillis(), TimeUnit.MILLISECONDS);
-            } finally {
-                try {
-                    watch.close();
-                } catch (IOException e) {
-                    LOG.debug("Error closing log reader.");
-                }
-            }
-        });
     }
 
     public static Predicate<HasMetadata> distictByResourceKey() {
@@ -500,6 +487,13 @@ public class OpenshiftProcessor {
             KubernetesClientErrorHandler.handle((KubernetesClientException) t);
         }
         return new RuntimeException("Execution of openshift build failed. See build output for more details", t);
+    }
+
+    private static void display(Reader logReader) throws IOException {
+        BufferedReader reader = new BufferedReader(logReader);
+        for (String line = reader.readLine(); line != null; line = reader.readLine()) {
+            LOG.info(line);
+        }
     }
 
     // visible for test

